.github/ISSUE_TEMPLATE/airflow_providers_bug_report.yml

```
@@ -0,0 +1,193 @@
+---
+name: Airflow Providers Bug report
+description: Problems and issues with code of Apache Airflow Provider
+labels: ["kind:bug", "area:providers"]
+body:
+  - type: markdown
+    attributes:
+      value: >
+        You do not need to create issue if you have a change ready to submit! You can opem
+        [Pull Request](https://github.com/apache/airflow/pulls) immediately.
+  - type: dropdown
+    attributes:
+      label: Apache Airflow Provider
+      description: Providers that the issue report is about (you can choose more than one)
+      multiple: true
+      options:
+        - airbyte
+        - alibaba
+        - amazon
+        - apache-beam
+        - apache-cassandra
+        - apache-drill
+        - apache-druid
+        - apache-hdfs
+        - apache-hive
+        - apache-kylin
+        - apache-livy
+        - apache-pig
+        - apache-pinot
+        - apache-spark
+        - apache-sqoop
+        - asana
+        - celery
+        - cloudant
+        - cncf-kubernetes
+        - databricks
+        - datadog
+        - dingding
+        - discord
+        - docker
+        - elasticsearch
+        - exasol
+        - facebook
+        - ftp
+        - google
+        - grpc
+        - hashicorp
+        - http
+        - imap
+        - jdbc
+        - jenkins
+        - jira
+        - microsoft-azure
+        - microsoft-mssql
+        - microsoft-psrp
+        - microsoft-winrm
+        - mongo
+        - mysql
+        - neo4j
+        - odbc
+        - openfaas
+        - opsgenie
+        - oracle
+        - pagerduty
+        - papermill
+        - plexus
+        - postgres
+        - presto
+        - qubole
+        - redis
+        - salesforce
+        - samba
+        - segment
+        - sendgrid
+        - sftp
+        - singularity
+        - slack
+        - snowflake
+        - sqlite
+        - ssh
+        - tableau
+        - telegram
+        - trino
+        - vertica
+        - yandex
+        - zendesk
+    validations:
+      required: true
+  - type: dropdown
+    attributes:
+      label: Apache Airflow version
+      description: >
+        What Apache Airflow version are you using? Only Airflow 2 is supported for bugs. If you wish to
+        discuss Airflow 1.10, open [Discussion](https://github.com/apache/airflow/discussions) instead!
+      multiple: false
+      options:
+        - "2.1.3 (latest released)"
+        - "2.1.2"
+        - "2.1.1"
+        - "2.1.0"
+        - "2.0.2"
+        - "2.0.1"
+        - "2.0.0"
+        - "main (development)"
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: Operating System
+      description: What Operating System are you using?
+      placeholder: "You can get it via `cat /etc/os-release` for example"
+    validations:
+      required: true
```

```suggestion
      required: true
  - type: textarea
    attributes:
      label: Versions of Apache Airflow Providers
      description: What Apache Airflow Providers versions are you using?
      placeholder: You can use `pip freeze | grep apache-airflow-providers` (you can leave only relevant ones)
```

We definitely want the provider version(s) here.

---

Right :)